rasa/utils/common.py

```diff
@@ -516,37 +516,6 @@ def directory_size_in_mb(
     return size / 1_048_576
 
 
-def copy_directory(source: Path, destination: Path) -> None:
-    """Copies the content of one directory into another.
-
-    Unlike `shutil.copytree` this doesn't raise if `destination` already exists.
-
-    # TODO: Drop this in favor of `shutil.copytree(..., dirs_exist_ok=True)` when
-    # dropping Python 3.7.
-
-    Args:
-        source: The directory whose contents should be copied to `destination`.
-        destination: The directory which should contain the content `source` in the end.
-
-    Raises:
-        ValueError: If destination is not empty.
```

Comment:
@radovanZRasa The function here raises a `ValueError` if the destination is not empty. However `shutil.copytree(..., dirs_exist_ok=True)` doesn't raise if the destination is not empty. 

These are not identical behaviour. What should I do here?